docs/README.md

```diff
@@ -140,7 +140,7 @@ private_key_filename: my-private-key
 
 ### Setup your customer access control list (ACL)
 
-Edit `customers_url` in gateway_config.yml.
```

Comment:
Maybe introduce each config file as it's referenced. Presume `gateway_config.yml` is the main configuration file for OpenFaaS cloud? and not related to the faas gateway in anyway